src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -192,6 +192,7 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
   // with this fragment if necessary. ActivityPlayer doesn't have knowledge about URL format and provided url_fragment 
   // to perform this merge automatically.
   const iframeUrl = activeDialog?.url || (embeddable.url_fragment ? url + embeddable.url_fragment : url);
+  const miContainerClass = questionNumber ? "managed-interactive has-question-number" : "managed-interactive";
```

Comment:
I'm assuming questionNumber is never 0, right?

Reply:
Yeah, it's either `undefined` or some number greater than zero.